src/annotator/highlighter.js

```diff
@@ -238,6 +265,8 @@ export function removeHighlights(highlights) {
 }
 
 /**
+ * Subset of `DOMRect` interface.
```

Comment:
We could potentially rewrite the `getBoundingClientRect` method below to just return the standard `DOMRect` type in future rather than have this additional `DOMRect`-like object.